core/src/main/java/org/infinispan/commands/write/ExceptionAckCommand.java

```diff
@@ -17,6 +21,7 @@
  * @since 9.0
  */
 public class ExceptionAckCommand extends BaseRpcCommand {
+   private static final Log log = LogFactory.getLog(ExceptionAckCommand.class);
```

Comment:
unused?

Reply:
Forgot about this comment, sorry.

Reply:
Me too, I've checked just tests and merged... let's not worry about that.